### ISSUE TITLE
Bug when trying to use CreateSteamWebInterface

### ISSUE THREAD

**Issue by sp0ok3r** [babelshift/SteamWebAPI2#83]
![error](https://user-images.githubusercontent.com/7952298/72682859-a20fbf80-3ac9-11ea-802d-132aa7816d5e.png)

**Comment by EAbychkov** [comment]
Same problem here..

**Comment by EAbychkov** [comment]
![image](https://user-images.githubusercontent.com/1067550/73187152-8785b800-4153-11ea-8a20-4be722f85d7d.png)

**Comment by babelshift** [comment]
Thanks. I'll take a look today.

**Comment by EAbychkov** [comment]
> Thanks. I'll take a look today.

Thank you for your prompt reply

**Comment by EAbychkov** [comment]
> Thanks. I'll take a look today.

How is it going? Any updates?

**Comment by babelshift** [comment]
Hey guys. Sorry, this repo is a side project for me when I get time outside my full time job (which can be very demanding). I'm trying to get some time soon. Thanks for understanding.

**Comment by babelshift** [comment]
@sp0ok3r @EAbychkov what version are you using? The latest (4.0.14)? What .NET runtime are you using? Full framework or .NET Core?

I'm having trouble reproducing this issue on .NET Core 2.1 console app with 4.0.14 nuget package. The following is working as expected in my setup.

```
var webInterfaceFactory = new SteamWebInterfaceFactory("<api_key>");
var steamInterface = webInterfaceFactory.CreateSteamWebInterface<PlayerService>(new HttpClient());
var playerSummaryResponse = await steamInterface.GetOwnedGamesAsync(<steam_id>);
```

**Comment by EAbychkov** [comment]
> @sp0ok3r @EAbychkov what version are you using? The latest (4.0.14)? What .NET runtime are you using? Full framework or .NET Core?
> 
> I'm having trouble reproducing this issue on .NET Core 2.1 console app with 4.0.14 nuget package. The following is working as expected in my setup.
> 
> ```
> var webInterfaceFactory = new SteamWebInterfaceFactory("<api_key>");
> var steamInterface = webInterfaceFactory.CreateSteamWebInterface<PlayerService>(new HttpClient());
> var playerSummaryResponse = await steamInterface.GetOwnedGamesAsync(<steam_id>);
> ```

.NET Core 3.0

**Comment by EAbychkov** [comment]
We are trying to use a code example from the main page:
var steamInterface = webInterfaceFactory.CreateSteamWebInterface<**SteamUser**>(new HttpClient());

**Comment by babelshift** [comment]
@EAbychkov the original screenshot in this issue from @sp0ok3r is using PlayerService.

Regardless, even when using SteamUser and migrating my console app .NET Core 3.1, I still cannot reproduce. Below is the full contents of the test class.

```
using System.Net.Http;
using System.Threading.Tasks;
using SteamWebAPI2.Interfaces;
using SteamWebAPI2.Utilities;

namespace TestSteamWebAPI2.Core
{
    class Program
    {
        async static Task Main(string[] args)
        {
            var webInterfaceFactory = new SteamWebInterfaceFactory("<api_key>");
            var steamInterface = webInterfaceFactory.CreateSteamWebInterface<SteamUser>(new HttpClient());
            var playerSummaryResponse = await steamInterface.GetPlayerSummaryAsync(76561197960361544);
            var playerSummaryData = playerSummaryResponse.Data;
        }
    }
}

```

Your screenshot shows a MethodNotFoundException in AutoMapper. What version of AutoMapper does your project use? SteamWebAPI2 relies on 6.1.1. Later versions are not guaranteed to work.

See: https://github.com/aspnetboilerplate/aspnetboilerplate/issues/4025 for an issue in AutoMapper 8 that looks identical to yours.

**Comment by EAbychkov** [comment]
@babelshift Do you have plans to move it to the AutoMapper 9? 
Because it makes it impossible to use. 
I have just moved your code to AutoMapper 9, it took about 20 min. But I want to use the NuGet package.

**Comment by babelshift** [comment]
Feel free to create a pull request with your changes.

**Comment by babelshift** [comment]
Updated AutoMapper #84 and Newtsonsoft.Json #85 and released on version 4.0.15.
